kafka-protocol-binding.md

```diff
@@ -246,12 +246,16 @@ content-type: application/cloudevents+json; charset=UTF-8
 
 {
     "specversion" : "1.0-rc1",
-    "datacontenttype" : "com.example.someevent",
+    "type" : "com.example.someevent",
+	"source" : "/mycontext/subcontext",
+	"id" : "1234-1234-1234",
+	"time" : "2018-04-05T03:56:24Z",
```

Comment:
The indentation is a bit off due to tabs instead of whitespace.

Reply:
done